modular_zubbers/code/modules/mob/living/carbon/human/species_types/felinid.dm

```diff
@@ -0,0 +1,10 @@
+/datum/species/human/felinid
+	use_skintones = FALSE
+	species_traits = list(
```

Comment:
This is defined on /datum/species as 	var/list/species_traits = list(HAS_FLESH,HAS_BONE) //by default they can scar and have bones/flesh unless set to something else

Reply:
Yes, you should add those.